test/fixtures/users.yml

```diff
@@ -115,3 +115,27 @@ system_admin:
   auth_source: internal
   password_hash: 02d7ff9921071af778ff4f8608579dcd6d80dfba
   password_salt: 80a167f1effbd82c2485ed81c3cfd68b11bc40dc
+
+disabled:
+  login: disabled
+  lower_login: disabled
+  firstname: Disabled
+  lastname: User
+  mail: disabled_user@someware.com
+  admin: fasle
+  disabled: true
+  last_login_on: 2009-10-12 21:50:04
+  auth_source: internal
+  password_hash: 3e4991bbc1df7cc5d43c8cf94838993a3045ba2a
+  password_salt: 852cefe7366f5c4e59276d2a753cba4302e44df3
+
+external_disabled:
```

Comment:
we do not have eny external user in fixtures (even though I don't like too much fixtures), so could make sense to have one, but we would need to have
```
auth_source: external
```
to trully be an external user, this one is external jut by the name...